lib/rex/post/sql/ui/console/command_dispatcher.rb

```diff
@@ -67,6 +69,15 @@ def unknown_command(cmd, line)
               super
             end
 
+            # @param [nil] cmd
+            # @param [Array] ignored
+            # @return [Object]
+            def cmd_help(cmd = nil, *ignored)
+              super
+
+              print_line(format_session_compatible_modules)
```

Comment:
This has the same problem as before;

```
PostgreSQL @ 127.0.0.1:9000 (template1) > help irb
Usage: irb

Open an interactive Ruby shell on the current session.

OPTIONS:

    -e          Expression to evaluate.
    -h, --help  Help menu.
This session also works with the following modules:
  auxiliary/admin/postgres/postgres_readfile
  auxiliary/admin/postgres/postgres_sql
  auxiliary/scanner/postgres/postgres_hashdump
  auxiliary/scanner/postgres/postgres_schemadump
  auxiliary/scanner/postgres/postgres_version
  exploit/linux/postgres/postgres_payload
  exploit/multi/postgres/postgres_copy_from_program_cmd_exec
  exploit/multi/postgres/postgres_createlang
  exploit/windows/postgres/postgres_payload

```

We don't want this list to always show